Bottom: 5f6f9448d389b0e7c9180b35c47a468134e77662
Top:    95648a7fd2db6864f376665a3aeddd27fcf3f532
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:03:32 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
